chapter_5/exercise_5_06/getline.c

```diff
@@ -1,23 +1,19 @@
 #include <stdio.h>
 
-void get_line(char *s);
 
-int main(void)
-{
-  char string[150] = "";
+char *getln(char *line);
 
-  get_line(string);
-  puts(string);
-
-  return 0;
+int main() {
+    char line[100];
```

Comment:
@kyopa The change from string to line in this context I think makes more sense. I see how this name better reflects the desired intention. Good job!